src/dmclock_server.h

```diff
@@ -1292,6 +1340,18 @@ namespace crimson {
       } // pull_request
 
 
+      void update_client_info(const C& client_id) {
```

Comment:
This function is repeated for both PullPriorityQueue and PushPriorityQueue. Shouldn't this be a public function in PriorityQueueBase?